src/test/java/teammates/test/cases/action/InstructorFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -35,15 +35,10 @@ protected String getActionUri() {
     }
 
     @Override
-    protected void prepareTestData() {
-        dataBundle = loadDataBundle("/InstructorFeedbackSubmissionEditSaveActionTest.json");
```

Comment:
@whipermr5 Can we declare two `dataBundle` here one for typical and one for `InstructorFeedbackSubmissionEditSaveActionTest.json`